brugge/config/assisted_history_matching.yml

```diff
@@ -64,34 +64,43 @@ model_parameters:
   fault_mult:
     min: 1.0e-9
     max: 1.0e-8
-    loguniform: True
+    loguniform: true
   relative_permeability:
-    scheme: global
+    scheme: regions_from_sim
+    interpolate: true
     regions:
       - id: None
         swirr:
           min: 0.0
+          base: 0.025
```

Comment:
I would suggest not to change Brugge per now - two reasons:

1. We haven't tested that the interpolation gives good results for Brugge.
2. We increase coverage to use different options for different cases. (Maybe a non-reason, as Egg also has that role)